### PR TITLE
Major/page banner refactor

### PR DESCRIPTION
**Pull request by eduardojmatos**
**CHANGELOG** :memo:

* Some refactors on page-banner;
* Removing --enveloped of picture;
* Adding siblings on menu--cover to adds padding on page-banner;
* Using variables on page-banner spaces.

**PRINTS** 🖼 

screen shot 2017-01-16 at 8 08 03 pm

### CODE REVIEW

File: docs/demo/gaiden-css/components/picture/cover.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  </style>
+</head>
+
+<body>
+  <picture class="picture picture--cover">
+    <source srcset="/demo/gaiden-css/images/background-header-desktop.jpg" media="(min-width: 1200px)">
+    <source srcset="/demo/gaiden-css/images/background-header-large.jpg" media="(min-width: 960px)">
+    <source srcset="/demo/gaiden-css/images/background-header-medium.jpg" media="(min-width: 768px)">
+    <source srcset="/demo/gaiden-css/images/background-header.jpg" media="(max-width: 767px)">
+    <img src="/demo/gaiden-cssimages/background-header-medium.jpg" class="picture__image" alt="Imagem">
```
Comment (thiagodelmotte) [inline docs/demo/gaiden-css/components/picture/cover.html]: @eduardojmatos não teria como descrever a imagem no alt, ou retirá-lo?

Comment (bsoares) [inline docs/demo/gaiden-css/components/picture/cover.html]: `gaiden-cssimages` => `gaiden-css/images`

File: docs/demo/gaiden-css/components/picture/demo.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  </style>
+</head>
+
+<body>
+  <picture class="picture">
+    <source srcset="/demo/gaiden-css/images/background-header-desktop.jpg" media="(min-width: 1200px)">
+    <source srcset="/demo/gaiden-css/images/background-header-large.jpg" media="(min-width: 960px)">
+    <source srcset="/demo/gaiden-css/images/background-header-medium.jpg" media="(min-width: 768px)">
+    <source srcset="/demo/gaiden-css/images/background-header.jpg" media="(max-width: 767px)">
+    <img src="/demo/gaiden-cssimages/background-header-medium.jpg" class="picture__image" alt="Imagem">
```
Comment (bsoares) [inline docs/demo/gaiden-css/components/picture/demo.html]: `gaiden-cssimages` => `gaiden-css/images`

File: docs/demo/gaiden-css/components/picture/rounded.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  </style>
+</head>
+
+<body>
+  <picture class="picture picture--rounded">
+    <source srcset="/demo/gaiden-css/images/background-header-desktop.jpg" media="(min-width: 1200px)">
+    <source srcset="/demo/gaiden-css/images/background-header-large.jpg" media="(min-width: 960px)">
+    <source srcset="/demo/gaiden-css/images/background-header-medium.jpg" media="(min-width: 768px)">
+    <source srcset="/demo/gaiden-css/images/background-header.jpg" media="(max-width: 767px)">
+    <img src="/demo/gaiden-cssimages/background-header-medium.jpg" class="picture__image" alt="Imagem">
```
Comment (bsoares) [inline docs/demo/gaiden-css/components/picture/rounded.html]: `gaiden-cssimages` => `gaiden-css/images`

File: docs/demo/gaiden-css/components/picture/cover.html (hunk 1 above)
Comment (eduardojmatos) [inline docs/demo/gaiden-css/components/picture/cover.html]: @thiagodelmotte this file is just a code sample. The objective of alt here is to force the importance of this attribute.

**Pull request merged**